cmd/delete/delete.go

```diff
@@ -20,5 +22,12 @@ func NewdeleteCmd() *cobra.Command {
 
 	socialCmd := social.NewsocialCmd()
 	cmd.AddCommand(socialCmd)
+
+	accountCmd := account.NewaccountCmd()
+	cmd.AddCommand(accountCmd)
+
+	account_uidCmd := account_uid.Newaccount_uidCmd()
+	cmd.AddCommand(account_uidCmd)
```

Comment:
Can't we use the same account command for both uid and email??
Something like
```
lr delete account --email // should delete account with given email
lr delete account --uid // should delete account with given uid
```
No need for separate cmd.